packages/enzyme-utilities/src/test/index.test.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import {mount} from 'enzyme';
+import {noop} from '@shopify/javascript-utilities/other';
```

Comment:
@TheMallen when did `javascript-utilities` get added to `quilt`? I think we should avoid using that directly, especially for something as simple as a `noop`. The plan is to deprecate `javascript-utilities` in favour of smaller packages in `quilt` AFAIK.